src/components/FilterItem/FilterItem.module.scss

```diff
@@ -6,4 +6,16 @@
   font-size: 1.1em;
   outline: none;
   text-transform: capitalize;
+  margin: 2px;
+}
+
+.selectedItem {
+  color: $color-pink;
+  &:hover {
+    color: $color-pink;
+  }
```

Comment:
does this hover make any sense? it's already pink

Reply:
Yes, because the common `button` style in `main.css` says that hover color is white. I have to overwrite it here.